external/gmock/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config GMOCK
+	bool "gmock"
```

Comment:
Are there any modules which is dependent to gmock?

Reply:
HAVE_CXX should be enabled.

Reply:
Then I think that adding dependency is better.
'depends on'